.github/workflows/pkg-pr-new.yml

```
@@ -0,0 +1,31 @@
+      - name: Install dependencies
+        run: deno install
+
+      - name: Build
+        run: deno task build
+
+      - name: Publish
+        run: deno run -A npm:pkg-pr-new publish --json output.json --comment=off
+        working-directory: npm
+
+      - name: Post or update comment
+        uses: ./.github/actions/comment-pkg-pr
```

actionlint (1.7.9)

file "dist/index.mjs" does not exist in "/home/jailuser/git/.github/actions/comment-pkg-pr". it is specified at "main" key in "runs" section in "Comment pkg-pr-new" action

(action)